src/Microsoft.Extensions.Testing.Abstractions/project.json

```diff
@@ -12,7 +12,8 @@
       "target": "project"
     },
     "Microsoft.DiaSymReader": "1.0.8",
-    "Microsoft.DiaSymReader.Native": "1.4.0-rc2"
+    "Microsoft.DiaSymReader.Native": "1.4.0-rc2",
+    "System.Reflection.Metadata": "1.4.1-beta-24410-02"
```

Comment:
This project already reference `Microsoft.DotNet.ProjectModel`, which references this package.  It seems unnecessary to duplicate the reference here.  The only thing we are doing is having another place to update when we need to update to a new version.